lib/jekyll_pages_api/page.rb

```diff
@@ -18,23 +21,31 @@ def filterer
     end
 
     def title
-      self.filterer.decode_html(self.page.data['title'] || '')
+      title = self.page.data['title'] if self.page.respond_to?(:data)
+      title = self.page.title if title == nil and self.page.respond_to?(:title)
```

Comment:
You can simplify this line to

```ruby
title ||= self.page.title if self.page.respond_to?(:title)
```

Are the conditionals to handle Pages vs. Posts etc? Might be worth checking for `is_a?`, or if you want to stick with duck-typing, at least adding a comment of the types that would be expected for each.